pyvista/examples/_dataset_loader.py

```diff
@@ -29,6 +29,7 @@
 """
 
 # ruff: noqa: PTH102,PTH103,PTH107,PTH112,PTH113,PTH117,PTH118,PTH119,PTH122,PTH123,PTH202
+# mypy: disable-error-code="redundant-expr"
```

Comment:
Disabled because of a very high number of `redundant-expr` and I am not very familiar to this new module. Might need some help if someone's interested, you are welcome :pray: